<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qwit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../qwit.qrc" >
    <normaloff>:/images/qwit.png</normaloff>:/images/qwit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="greetingMessageLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>What are you doing?</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="leftCharactersNumberLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>140</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lastStatusLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="midLineWidth" >
        <number>3</number>
       </property>
       <property name="text" >
        <string>watching movie - 10 minutes ago</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_3" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="tabPosition" >
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <property name="elideMode" >
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="homeTab_2" >
        <attribute name="title" >
         <string>Home</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Your and your friends' timeline</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="publicTab_2" >
        <attribute name="title" >
         <string>Public</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Public timeline</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="repliesTab_2" >
        <attribute name="title" >
         <string>Replies</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Replies to you</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="customTab_2" >
        <attribute name="title" >
         <string>Custom</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Custom user timeline</string>
        </attribute>
        <layout class="QGridLayout" name="_8" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" name="_9" >
           <item>
            <layout class="QHBoxLayout" name="_10" >
             <item>
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>Username:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="customUsernameLineEdit_2" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="inboxTab_2" >
        <attribute name="title" >
         <string>Inbox</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Direct messages inbox</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="outboxTab_2" >
        <attribute name="title" >
         <string>Outbox</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Direct messages outbox</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="searchTab_2" >
        <attribute name="title" >
         <string>Search</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Search for messages</string>
        </attribute>
        <layout class="QGridLayout" name="_11" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" name="_12" >
           <item>
            <layout class="QHBoxLayout" name="_13" >
             <item>
              <widget class="QLabel" name="label_5" >
               <property name="text" >
                <string>Search:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchLineEdit_2" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stateLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="indent" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>26</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Refresh</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
         </property>
         <property name="shortcut" >
          <string>Ctrl+R</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optionsPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>26</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Options</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logsPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>26</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/logs.png</normaloff>:/images/logs.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitPushButton" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>26</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Quit</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qwit.qrc" />
  <include location="../qwit.qrc" />
 </resources>
 <connections/>
</ui>
